src/main/resources/META-INF/rewrite/jakarta-ee-9.yml

```
@@ -218,6 +230,12 @@ recipeList:
       newGroupId: jakarta.authorization
       newArtifactId: jakarta.authorization-api
       newVersion: latest.release
+  - org.openrewrite.java.dependencies.ChangeDependency:
+      oldGroupId: org.glassfish.main
+      oldArtifactId: javax.security.jacc
+      newGroupId: jakarta.authorization
+      newArtifactId: jakarta.authorization-api
+      newVersion: latest.release
```

Similarly, following redirects: https://mvnrepository.com/artifact/org.glassfish.main/javax.security.jacc
```suggestion
  - org.openrewrite.java.dependencies.ChangeDependency:
      oldGroupId: org.glassfish.main
      oldArtifactId: javax.security.jacc
      newGroupId: jakarta.security.jacc
      newArtifactId: jakarta.security.jacc-api
      newVersion: latest.release
```
But again: mvnrepository only shows a single version, so perhaps that meta data can't be trusted.

---

Same case as for javax.security.auth.message

---

Similarly resolved based on the contents of the Jar returned by
https://mvnrepository.com/artifact/jakarta.authorization/jakarta.authorization-api/3.0.0